src/main/java/opendcs/dao/ComputationDAO.java

```diff
@@ -252,11 +252,11 @@ public DbComputation getComputationById(DbKey compId) throws DbIoException,
 		if (ret != null)
 			return ret;
 
-		try(
-			PreparedStatement getComp = getConnection().prepareStatement(
+		try(Connection c = getConnection();
+			PreparedStatement getComp = c.prepareStatement(
 				"select " + compTableColumns + " from CP_COMPUTATION where COMPUTATION_ID = ?"
 			);
-			PreparedStatement getAppId = getConnection().prepareStatement(
+			PreparedStatement getAppId = c.prepareStatement(
 				"select LOADING_APPLICATION_NAME from HDB_LOADING_APPLICATION where LOADING_APPLICATION_ID = ?"
```

Comment:
Consider a future refactor  of the getAppId query into HdbComputation?

Reply:
Think this table is OpenDCS-wide?

Reply:
yep. the tablename kept hdb in it, we *could* change it to "loading_applcation" but honestly then we'd just have to have logic in place to detect the right name for backwards compatibility and it's still there anyways.

Maybe that can be an 8.0 thing.